src/gobmanagement/grpc/README.md

```diff
@@ -0,0 +1,24 @@
+# gRPC
+
+The ```grpc``` directory contains three main subdirectories: ```out```,
+```protos``` and ```services```, plus a ```codegen.py``` file in the
+root.
+
+The ```out``` and ```protos``` directory contain files that are
+generated by ```codegen.py```.  The proto file in the ```protos```
+directory is generated from the StartCommands defined in GOB Core.
+From that, the Python skeleton code in ```out``` is generated using the
+gRPC proto compiler.
+
+The ```services``` directory contains our implementation.
+
+## Running codegen.py
+```codegen.py``` only needs to be run after the a change is made in the
+start commands.
+
```

Comment:
Why not make this part of the build process?

Reply:
Included now :)